site/app/exceptions/AggregateException.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace app\exceptions;
+
+/**
+ * Class AggregateException
+ * @package app\exceptions
+ *
+ * When multiple separate errors must be communicated through one exception
+ */
+class AggregateException extends BaseException {
```

Comment:
Given how you're using this, you should probably just call this "ValidationException" which better reflects that it's being used to point out validation errors.